### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards- Home]: Talkback does not announce the role and position of the tabs present under home screen.

### ISSUE THREAD

**Issue by Bhaskar301101** [microsoft/AdaptiveCards#8418]
### Target Platforms

Android

### SDK Version

2023.01.20.1

### Application Name

AdaptiveCards

### Problem Description

**Test Environment:**
OS: Android 13
Device: S21+ 5G
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.01.20.1

**Repro Steps:**
1. Turn on Talkback.
2. Open Adaptive Cards Visualizer Application.
3. Home Page will opened.
4. Now Navigate to "import, visual, json, conf." tabs using swipe gesture and observe the issue.

**Actual Result:**
Talkback does not announce the role and position of the tabs present under home screen, Talkback announce like " Import selected, Visual double tap to activate".

**Expected Result:**
Talkback should announce the role and position of the tabs present under home screen, Talkback should announce like " Import tab selected, Visual tab, 2 of 4 double tap to activate".

**User Experience:**
Screen reader users will be impacted as they will be unable to understand whether the controls are interactive or not and if they are interactive then whether they are buttons or links or tabs.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/89897257/228434792-7dda0871-0ec6-4f6e-8a0b-322a37d41d79.mp4

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
we don't fix a11y issues on sample app since we don't ship it to the public

**Comment by vagpt** [comment]
As per the above comment closing this issue.
